.github/workflows/multichain-e2e.yml

```
@@ -121,3 +122,12 @@ jobs:
       config: config.xcs-swap-anything.yaml
       test_command: yarn test:xcs
       test_suite_name: swap-on-osmosis
+
+  ymax-hermes:
+    name: YMax - Hermes
+    if: github.event_name == 'workflow_dispatch' && inputs.test_type == 'ymax-hermes'
+    uses: ./.github/workflows/multichain-e2e-ymax.yml
```

No need for another file... `multichain-e2e-template.yml` will do:

```suggestion
    uses: ./.github/workflows/multichain-e2e-template.yml
```